### DISCUSSION TITLE
german model trained on thorsten dataset poorly deals with newline characters

### DISCUSSION THREAD

**Discussion by 54696d21** [coqui-ai/TTS#416] [General]
newline characters in the input text result in the model creating babbling nonsense or just repeating a word over and over. (I know that's easy to fix by removing newlines from the input) Is it possible to make the model treat a newline like a space character?

**Comment by erogol** [comment]
Do you mean this \n?

Model tries to read every character it is given so if you have special characters you need to stip them in advance.

**Reply by 54696d21** [reply]
yes, I mean \n

Isn't there a way to train the model in a way to treat \n as a space character?

**Reply by erogol** [reply]
you can add /n in sentences and train the model with it so that the model can learn to ignore
